### ISSUE TITLE
No text rendering, login not possible

### ISSUE THREAD

**Issue by pointhi** [Antergos/lightdm-webkit2-greeter#33]
Probably releated to #31

After updating, I'm not longer able to login, because:
* there is no text rendered, as well as checkboxes, input fields,...
* when I do the same gestures as with the old theme, I'm still not able to login

Correctly rendered are:
* the login rect (the big one containing the Antergos logo and which normaly also includes all other input fields)
* the antergos logo
* a little arrow in the login rect which can be clicked (but I don't know what it do, it simply changes position after clicking)
* the menue at the right upper position. But when I click it, I only see entries like ```{{i18n-entry}}``` (something like this)
* the mouse pointer (not in all cases, I only see either the normal mouse pointer or the hand symbol)

Not rendered are:
* The Background (probably see #31 )
* All Fonts (I see no text (except the ```{{i18n-entry}}``` strings of the upper right menue
* Input field, button to press login/which is shown for incorrect login

```
$ cat /etc/lightdm/lightdm-webkit2-greeter.conf
#
# background = Background file to use, either an image path or a color (e.g. #772953)
# theme-name = GTK+ theme to use
# font-name = Font to use
# xft-antialias = Whether to antialias Xft fonts (true or false)
# xft-dpi = Resolution for Xft in dots per inch (e.g. 96)
# xft-hintstyle = What degree of hinting to use (hintnone, hintslight, hintmedium, or hintfull)
# xft-rgba = Type of subpixel antialiasing (none, rgb, bgr, vrgb or vbgr)
#
[greeter]
#background=/usr/share/backgrounds/warty-final-ubuntu.png
background=/usr/share/backgrounds/gnome/adwaita-lock.jpg
theme-name=Ambiance
webkit-theme=antergos
#font-name=Ubuntu 11
font-name=DejaVuSans 11
xft-antialias=true
xft-dpi=96
xft-hintstyle=slight
xft-rgba=rgb


#[greeter]
#background=#000000
#theme-name=Adwaita
#webkit-theme=antergos
#font-name=DejaVuSans 11
#xft-antialias=true
#xft-dpi=96
#xft-hintstyle=slight
#xft-rgba=rgb
```

In the beginnig, there was the second greeter enabled. This is my changed version to pinpoint the bug, but they changed nothing of the appearance of lightdm

```
$ cat /var/log/lightdm/x-0-greeter.log
undefined: CONSOLE ERROR Die Schlüsselwertedatei enthält nicht die Gruppe »branding«
```

I'm using Antergos with Gnome 3.20 on X-Server

**Comment by pointhi** [comment]
ok, pacman didn't updated ```/etc/lightdm/lightdm-webkit2-greeter.conf```, copying the new version solved the main issue.

Nevertheless, a old config file shouldn't cause a not usable login screen, so I leave this issue open.

**Comment by lots0logs** [comment]
This has been fixed in 2.1.1
